metriq-gym/v0.5/quantinuum/h2-2/2025-12-08_09-24-14_qml_kernel_30507a53.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "app_version": "0.4.3.dev12+g1b4196a92",
```

Comment:
This should be 0.5 at this point (just like in the path of the file) 🤔 
Probably a `uv sync` is needed for that to happen.

Reply:
I just did a 
```
$ uv sync
$ uv run mgym job poll 697d7984-ea3c-4642-bab0-129f978a09c3 --no-cache
{'app_version': '0.4.3.dev12+g1b4196a92',
 'job_type': 'QML Kernel',
 'platform': {'device': 'H2-2',
              'device_metadata': {'simulator': False, 'version': '0.54.0'},
              'provider': 'quantinuum'},
 'results': {'accuracy_score': {'uncertainty': 0.013034991369387246,
                                'value': 0.783},
             'score': {'uncertainty': 0.013034991369387246, 'value': 0.783}},
 'suite_id': None,
 'timestamp': '2025-12-08T09:24:14.826067'}
```

Is this because the job was initially submitted with that other version even if I'm polling/uploading with 5.0?